ext/standard/basic_functions.stub.php

```diff
@@ -600,50 +600,50 @@ function str_starts_with(string $haystack, string $needle): bool {}
 
 function str_ends_with(string $haystack, string $needle): bool {}
 
-function chunk_split(string $str, int $chunklen = 76, string $ending = "\r\n"): string {}
+function chunk_split(string $string, int $length = 76, string $separator = "\r\n"): string {}
 
-function substr(string $str, int $start, ?int $length = null): string {}
+function substr(string $string, int $start, ?int $length = null): string {}
 
-function substr_replace(array|string $str, array|string $replace, array|int $start, array|int|null $length = null): string|array {}
+function substr_replace(array|string $string, array|string $replace, array|int $start, array|int|null $length = null): string|array {}
 
-function quotemeta(string $str): string {}
+function quotemeta(string $string): string {}
 
 function ord(string $character): int {}
 
 function chr(int $codepoint): string {}
 
-function ucfirst(string $str): string {}
+function ucfirst(string $string): string {}
 
-function lcfirst(string $str): string {}
+function lcfirst(string $string): string {}
 
-function ucwords(string $str, string $delimiters = " \t\r\n\f\v"): string {}
+function ucwords(string $string, string $delimiters = " \t\r\n\f\v"): string {}
 
-function strtr(string $str, string|array $from, ?string $to = null): string {}
+function strtr(string $string, string|array $from, ?string $to = null): string {}
```

Comment:
Slightly unrelated: I saw that the manual has a very detailed description about the two versions of `strtr()` (https://www.php.net/manual/en/function.strtr.php), and this is the first time when I realized that the default value fix for PHP 8.0 has a side-effect that we can't have an entirely accurate name for the 2nd parameter :/

Reply:
Something to discuss, but I think what we should do with these overloaded signatures is to keep listing them as multiple signatures in the manual and define a policy that overloaded functions are excluded from our parameter name stability policy -- and that we reserve the right to make invoking them with named params a hard error in the future (once we have "positional only" arguments). We just can't support these kinds of signatures properly.